<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveJobInfoDlg</class>
 <widget class="QDialog" name="ReceiveJobInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WRX - Job Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>activated</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="labActivated">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Name in result file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Start  UTC</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Duration  in sec</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Time conflict</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Frequencies</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QTableWidget" name="twgFrequencies">
     <column>
      <property name="text">
       <string>Call-Sign</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Frequency</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mode</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modulation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Radio</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>651</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="labName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="labId">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="labFileName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="labStartUTC">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="labDuration">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLabel" name="labTimeConflict">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveJobInfoDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>869</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
